CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
   - Add CA compliance notice of collection
+  - Venmo: add error message prompting the customer to click the Venmo button when `requestPaymentMethod` is called. (closes \#882)
```

Comment:
we usually link to the issue, does this syntax do that, too?

Reply:
I thought it did but I guess not! Fixed it, so it should link now.